app/src/main/java/com/example/asiantech/playmusic/MainActivity.java

```diff
@@ -0,0 +1,453 @@
+package com.example.asiantech.playmusic;
+
+import android.app.ActivityManager;
+import android.content.BroadcastReceiver;
+import android.content.ComponentName;
+import android.content.ContentResolver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.ServiceConnection;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.Handler;
+import android.os.IBinder;
+import android.provider.MediaStore;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentActivity;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v4.view.ViewPager;
+import android.util.Log;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.SeekBar;
+import android.widget.TextView;
+
+import com.astuetz.PagerSlidingTabStrip;
+import com.example.asiantech.playmusic.adapter.PagerAdapter;
+import com.example.asiantech.playmusic.fragment.PlaySongFragment_;
+import com.example.asiantech.playmusic.fragment.SongListFragment_;
+import com.example.asiantech.playmusic.model.Song;
+import com.example.asiantech.playmusic.service.MusicService;
+
+import org.androidannotations.annotations.AfterViews;
+import org.androidannotations.annotations.Click;
+import org.androidannotations.annotations.EActivity;
+import org.androidannotations.annotations.ViewById;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Formatter;
+import java.util.Locale;
+
+import lombok.Getter;
+
+@EActivity(R.layout.activity_main)
+public class MainActivity extends FragmentActivity implements OnItemListener {
```

Comment:
WARNING: [Registered] The `<activity> com.example.asiantech.playmusic.MainActivity` is not registered in the manifest
 Activities, services and content providers should be registered in the `AndroidManifest.xml` file using `<activity>`, `<service>` and `<provider>` tags.

If your activity is simply a parent class intended to be subclassed by other "real" activities, make it an abstract class.